xbmc/threads/platform/pthreads/ThreadImpl.cpp

```diff
@@ -61,6 +61,8 @@ static pid_t GetCurrentThreadPid_()
   return gettid();
 #elif TARGET_DARWIN_TVOS
```

Comment:
I'd just use this for all Darwin targets:

```suggestion
 #elif TARGET_DARWIN
```

Reply:
Ok